doc/sphinx/Pacemaker_Development/components.rst

```diff
@@ -169,18 +169,18 @@ messaging layer callback, which calls:
 
   * ``handle_reply()``, which calls
 
-    * ``process_remote_stonith_exec()``, which calls either
-      ``call_remote_stonith()`` (to retry a failed operation, or try the next
-       device in a topology is appropriate, which issues a new
+    * ``fenced_process_fencing_reply()``, which calls either
+      ``request_peer_fencing()`` (to retry a failed operation, or try the next
+      device in a topology is appropriate, which issues a new
       ``STONITH_OP_FENCE`` request, proceeding as before) or
-      ``remote_op_done()`` (if the operation is definitively failed or
+      ``finalize_op()`` (if the operation is definitively failed or
       successful).
 
-      * remote_op_done() broadcasts the result to all peers.
+      * ``finalize_op()`` broadcasts the result to all peers.
 
 Finally, all peers receive the broadcast result and call
 
-* ``remote_op_done()``, which sends the result to all local clients.
+* ``finalize_op()``, which sends the result to all local clients.
 
 
```

Comment:
Would this be a good place to add some info where in that flow history
is updated / how updates get distributed to the hosts / how hosts having
been disconnected from (quorate part of ) the cluster get synced up /
where notification-callbacks are triggered?
Not sure which parts of that info should be woven into the above making
it less readable but more complete or if we should keep history and
notification documentation totally separate.

Reply:
Good point. I think a separate section for how history works would be good